dev/files-repo/terraform.tfvars

```diff
@@ -14,8 +14,10 @@ uri_map = {
   "https://archive.repo.saltstack.com/windows/Salt-Minion-2019.2.5-Py3-AMD64-Setup.exe" = "saltstack/salt/windows/"
   "https://archive.repo.saltstack.com/windows/Salt-Minion-2019.2.8-Py2-AMD64-Setup.exe" = "saltstack/salt/windows/"
   "https://archive.repo.saltstack.com/windows/Salt-Minion-2019.2.8-Py3-AMD64-Setup.exe" = "saltstack/salt/windows/"
-  "https://repo.saltstack.com/windows/Salt-Minion-3003-Py3-AMD64-Setup.exe"             = "saltstack/salt/windows/"
-  "https://repo.saltstack.com/windows/Salt-Minion-3003.3-Py3-AMD64-Setup.exe"           = "saltstack/salt/windows/"
+  "https://repo.saltproject.io/windows/Salt-Minion-3004-2-Py3-AMD64-Setup.exe"          = "saltstack/salt/windows/"
+  "https://repo.saltproject.io/windows/Salt-Minion-3003.3-Py3-AMD64-Setup.exe"          = "saltstack/salt/windows/"
+  "https://repo.saltproject.io/windows/Salt-Minion-3003-Py3-AMD64-Setup.exe"            = "saltstack/salt/windows/"
+  
```

Comment:
I believe the linter is asking that this line be removed. But you can run `terraform fmt -recursive` to be sure...